e2etest/test_manager_btcstaking.go

```diff
@@ -50,12 +50,10 @@ func (tm *TestManager) getBTCUnbondingTime(t *testing.T) uint32 {
 	btccParams, err := tm.BabylonClient.BTCCheckpointParams()
 	require.NoError(t, err)
 
-	m := sdkmath.Max[uint32](
-		bsParams.Params.MinUnbondingTimeBlocks,
+	return sdkmath.Max[uint32](
+		bsParams.Params.UnbondingTimeBlocks,
 		btccParams.Params.CheckpointFinalizationTimeout,
```

Comment:
we can get unbonding time directly from params without comparing it with  checkpoint finalization timeout